src/js/library.js

```diff
@@ -0,0 +1,12 @@
+export default function createElement(elementType, classNames = '', attributes = {}, innerHTML = '', properties = {}) {
+  const elementObject = document.createElement(elementType);
+  if (classNames) elementObject.classList.add(...(classNames.split(' ')));
+  Object.keys(attributes).forEach((attribute) => {
+    elementObject.setAttribute(attribute, attributes[attribute]);
+  });
+  Object.keys(properties).forEach((property) => {
+    elementObject[property] = properties[property];
+  });
+  elementObject.innerHTML = innerHTML;
+  return elementObject;
+}
```

Comment:
- [ ] As I explained in the above statement, kindly use `arrow` functions instead of the `function` keyword.